onboarding.md

```diff
@@ -0,0 +1,42 @@
+## Welcome to SIG-Security team 👋
+
+## Onboarding Tasks
+
+Alright! Time to get our hands dirty 👷 Work through and get acquainted with the process. 
+<br>Please reach out to the team if you need any support 😃
+
+## Lets get going!
+As you walk through this, please remember that it is a living checklist and may require updates or corrections. You can propose updates to the template by editing it under [Onboarding](onboarding.md). 
+
+- [ ] Enable 2FA on GitHub
+- [ ] Review the [Code of Conduct](CODE_OF_CONDUCT.md)
+- [ ] Skim through [Contributing to Eclipse Tractus-X](CONTRIBUTING.md)
+- [ ] Create an Eclipse Foundation Account - [Sign Up](https://accounts.eclipse.org/user/register)
+  - [ ] Follow the guidelines from [Tractus-X Eclipse Foundation Rules](https://eclipse-tractusx.github.io/docs/oss/getting-started)
+  - [ ] Subscribe to Eclipse Foundation [mailing-list](https://accounts.eclipse.org/mailing-list/tractusx-dev)
+  - [ ] Create a new issue under sig-security to be added as member for legitimization and wait for the approval. <br>[Onboard a new member to join SIG-Security](https://github.com/eclipse-tractusx/sig-security/issues/new/choose).
+  - [ ] Request any committer from SIG-security to add you to the Eclipse Tractus-X GitHub organisation as a contributor or use [GitHub Issue Template](https://github.com/eclipse-tractusx/sig-infra/issues/new?assignees=&labels=support&projects=&template=support-add-project-contributor.md&title=New+Tractus-X+project+contributor)
```

Comment:
Contributors can be added by any committer. Not sure if you want to explicitly keep it to sig-security members

Reply:
Yes, we also decided not to keep it specific for sig-security members.